### DISCUSSION TITLE
Re-add default catalog for new icons?

### DISCUSSION THREAD

**Discussion by xEtarusx** [TomBursch/kitchenowl#169] [Q&A]
Hi there,
is there any possibility to re-add the default item catalog for an existing household?

Only new households have the new icons in the catalog by default. I know I could add icons by myself to each and every icon but it would be nice to have at least the default ones which now have icons.

With existing households all items were not "upgraded" to get it's respective item, even when they weren't changed at all from the default catalog.

Thanks for any ideas
Etarus

**Comment by TomBursch** [comment]
I'm working on an automatic way to keep default items up to date. For now, you could set the household language in the household settings, which should add all new default items and set the icons for existing ones. Though not all default items have icons assigned yet.

**Reply by xEtarusx** [reply]
Setting the language again in the settings did work. Thank you for the hint.
Looking forward to any automation for updating the catalog 😃

**Reply by jakejoh** [reply]
is it possible, that chaning the language stopped working? I remember doing that, but can't find a way to do with v0.5.1/96
